portfolio/src/main/webapp/script.js

```diff
@@ -94,4 +94,27 @@ function changeDropdownVal() {
     const dropdown = document.getElementById("num-comments");
     localStorage["num-comments"] = dropdown.value;
     this.form.submit();
+}
+
+function deleteMessages() {
+    console.log("Fetching response.");
+    const responsePromise = fetch("/delete-data", {method:"POST"});
```

Comment:
This can also be a DELETE rather than a POST since it doesn't require any form data, but I think this is fine for now.